package/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.suse.com/bci/golang:1.23 AS builder
+RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.56.2
+WORKDIR /usr/src/app
+COPY ../go.mod go.sum ./
+RUN go mod download
+COPY .. .
+RUN make ci
+
+FROM registry.suse.com/bci/bci-micro:latest
+COPY --from=builder /usr/src/app/bin/pushprox-proxy /usr/bin
+COPY --from=builder /usr/src/app/bin/pushprox-client /usr/bin
```

Comment:
I think we may be wise to include a default entrypoint script with this.  Strictly from a defensive POV to provide direction that the image should be started with one of the 2 specific supported entry points. I know our chart use case shouldn't allow this to happen - but customers do weird things.